### ISSUE TITLE
Add ARIA roles and attributes to the tab navigation component

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#159]
### Problem

[molecules/molecules.navigation](https://a-ui.github.io/core_branding_scss/#molecules-navigation)

The markup of the tab navigation is missing ARIA in order to work properly with assitive technologies such as screenreaders.

### Solution

Follow the ARIA design pattern for tabs.

### Articles and links

[ARIA design pattern: Tabs](https://www.w3.org/TR/wai-aria-practices-1.2/#tabpanel)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)
